### PR TITLE
Integrate Travis to do full deployment

### PR DESCRIPTION
**Pull request by meganle**
- Used Travis to encrypt pem key
- After docker image push, ssh into the server and set `IMAGE_TAG` to the commit hash
- Run docker stack deploy

Edited docker-compose.yml on server to use env variable`IMAGE_TAG` instead of just copying in the commit hash:
screen shot 2018-11-17 at 2 55 23 pm

Successfully tested deployment for my branch through Travis:
screen shot 2018-11-17 at 2 55 45 pm

Travis log can be found here (under Deploying Application): https://travis-ci.org/cuappdev/pollo-backend/builds/456433762?utm_source=github_status&utm_medium=notification

### CODE REVIEW

File: docker_push [hunk 1]
```diff
@@ -3,4 +3,6 @@ echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
 docker build -t pollo --no-cache .
 docker tag $(docker images -q pollo) cornellappdev/pollo:"$TRAVIS_COMMIT"
 docker push cornellappdev/pollo:"$TRAVIS_COMMIT"
-
+chmod 600 server.pem
+ssh -i server.pem appdev@pollo-backend.cornellappdev.com "cd deploy/docker-compose; export IMAGE_TAG='$TRAVIS_COMMIT';
```
Comment (young-k) [inline docker_push]: i don't think pollo needs `registry-auth`, since it's not using a private registry?

Comment (young-k) [inline docker_push]: we should standardize this, the latest version of `swarm-cli` puts the files in `~/docker-compose`, so we should standardize that here as well

File: docker_push [hunk 2]
```diff
@@ -3,4 +3,6 @@ echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
 docker build -t pollo --no-cache .
 docker tag $(docker images -q pollo) cornellappdev/pollo:"$TRAVIS_COMMIT"
```
Comment (young-k) [inline docker_push]: why not just tag it with `cornellappdev/pollo:"$TRAVIS_COMMIT"` instead of doing it in two steps?

File: .travis.yml [hunk 3]
```diff
@@ -1,25 +1,30 @@
 language: node_js
 sudo: required
 node_js:
-    - '6'
+- '10'
 services:
-    - postgresql
-    - docker
+- postgresql
+- docker
```
Comment (young-k) [inline .travis.yml]: alphabetize

**Comment by meganle** [comment]
Yep, I commented the Travis build along with a screenshot of testing the deployment on my branch.

**Pull request merged**